### ISSUE TITLE
alpha in batch algorithm for SOM

### ISSUE THREAD

**Issue by FranciscoRMendes** [macss-modeling/General-Questions#31]
I was trying to change the $\alpha$ parameter i.e. the learning rate for the SOM based on this code chunk shared in class :
`som_fit <- som(anes_scaled,
               grid = search_grid,
               alpha = c(0.1, 0.001), 
               radius = 1, 
               rlen = 500, 
               dist.fcts = "euclidean", 
               mode = "batch") `
However, it seems that the documentation says that $alpha$ is not used for the batch algorithm. When I change the alpha rate the convergence does not speed up or slow down. When I did a Google search for the updating formula in Kohonen's 92 paper I see that : 
![image](https://user-images.githubusercontent.com/19298622/110439445-7c82b280-80dd-11eb-86d1-81188f74b1bd.png)
This equation does not have $\alpha$
Can you please tell me whether the hyperparameter alpha is used in the batch algorithm?

**Comment by pdwaggoner** [comment]
`\alpha` is not used in batch versions, but is in online algs. E.g., in another implementation from the `class` package, note the lack of \alpha in in the batchSOM() function (https://www.rdocumentation.org/packages/class/versions/7.3-18/topics/batchSOM). But by default, `\alpha` ranges for the `kohonen` implementation as well as the `SOM()` implementation from `class` (e.g., https://www.rdocumentation.org/packages/class/versions/7.3-18/topics/SOM). So if this is included, even when calling `batch`, it won't mess up computation (or help for that matter).
